src/content/docs/kubernetes-pixie/kubernetes-integration/understand-use-data/find-use-your-kubernetes-data.mdx

```diff
@@ -10,10 +10,13 @@ metaDescription: 'New Relic''s Kubernetes integration: How to install and activa
 redirects:
   - /docs/integrations/kubernetes-integration/understand-use-data/find-use-your-kubernetes-data
   - /docs/integrations/kubernetes-integration/understand-use-data/understand-use-data
+  - /docs/integrations/kubernetes-integration/installation/install-kubernetes-integration-using-helm/#services
 ---
 
 import kubernetesDashboard from 'images/kubernetes_screenshot-full_dashboard.png'
 
+After having installed our Kubernetes integration, you can start instrumenting the services that run in your cluster. To learn more about how to do this, please check our [Monitor services running on Kubernetes](/docs/integrations/kubernetes-integration/link-apps-services/monitor-services-running-kubernetes/#monitor-services-helm) page and our detailed [Find and use your Kubernetes data](/docs/integrations/kubernetes-integration/understand-use-data/) pages.
```

Comment:
Monitoring services inside k8s is known to PMs to be a very rough experience at the moment. There are some efforts ongoing to rethink and simplify how it's done, but for now I would suggest to avoid suggesting it as a part of the recommended journey.

Reply:
Removed this part: "start instrumenting the services that run in your cluster. To learn more about how to do this, please check our [Monitor services running on Kubernetes](/docs/integrations/kubernetes-integration/link-apps-services/monitor-services-running-kubernetes/#monitor-services-helm) page and"